src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -56,6 +56,24 @@ public RangeConstraint(IComparable from, IComparable to) : base( from, to )
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="RangeConstraint"/> class for objects that did not implement <see cref="IComparable"/>.
+        /// </summary>
+        /// <remarks>from must be less than or equal to true</remarks> 
+        /// <param name="from">Inclusive beginning of the range. Must be less than or equal to to.</param>
+        /// <param name="to">Inclusive end of the range. Must be greater than or equal to from.</param>
+        /// <param name="comparer">Class that implements <seealso cref="IComparer"/> used to compare the objects.</param>
+        public RangeConstraint(object from, object to, IComparer comparer) : base(from, to)
+        {
+            // from must be less than or equal to to
```

Comment:
Not sure why this comment is needed, but it contradicts the exception message just below.